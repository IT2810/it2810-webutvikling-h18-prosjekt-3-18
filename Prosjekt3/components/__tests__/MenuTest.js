import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Menu from '../../components/menu';

it('renders correctly', () => {
    const tree = renderer.create(<Menu>Snapshot test!</Menu>).toJSON();
    expect(tree).toMatchSnapshot();
});
