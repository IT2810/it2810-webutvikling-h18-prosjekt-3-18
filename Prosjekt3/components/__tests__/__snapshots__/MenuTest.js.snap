// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "paddingTop": 22,
    }
  }
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
       
       
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      style={
        Object {
          "borderTopWidth": 2,
          "height": 40,
          "marginTop": 20,
        }
      }
      value="Homework"
    />
    <View
      accessibilityComponentType="button"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          Add
        </Text>
      </View>
    </View>
  </View>
</View>
`;
